app/src/main/java/net/gsantner/markor/ui/hleditor/HighlightingEditor.java

```diff
@@ -238,25 +238,17 @@ public void simulateKeyPress(int keyEvent_KEYCODE_SOMETHING) {
         dispatchKeyEvent(new KeyEvent(0, 0, KeyEvent.ACTION_UP, keyEvent_KEYCODE_SOMETHING, 0));
     }
 
-    public void insertOrReplaceTextOnCursor(String newText) {
-        newText = (newText == null ? "" : newText);
-        int newCursorPos = newText.indexOf(PLACE_CURSOR_HERE_TOKEN);
-        newText = newText.replace(PLACE_CURSOR_HERE_TOKEN, "");
-        int start = Math.max(getSelectionStart(), 0);
-        int end = Math.max(getSelectionEnd(), 0);
-        getText().replace(Math.min(start, end), Math.max(start, end), newText, 0, newText.length());
-        if (newCursorPos >= 0) {
-            setSelection(start + newCursorPos);
-        }
-    }
-
-    public int getShiftWidth(String text) {
-        if (text.contains("sw=2") || text.contains("shiftwidth=2")) {
-            return 2;
-        } else if (text.contains("sw=8") || text.contains("shiftwidth=8")) {
-            return 8;
-        } else {
```

Comment:
getShiftWidth and any sw= mentions are just removed. right?

Reply:
Yeah. We don't use this anywhere. Shift is set per-file by long-pressing the indent button